projects/Apigen/apigen/Apigen.cs

```diff
@@ -1112,6 +1112,10 @@ public string SanitisedFullName(Type t)
             {
                 return "IDictionary<string, object>";
             }
+            if (t.FullName.StartsWith("System.ReadOnlyMemory`1[[System.Byte"))
```

Comment:
Is there really no better way to distinguish between a byte array and `ReadOnlyMemory`?

Reply:
This is probably the "easiest" way to compare `t` with a known generic type. Comparing the `t` instance directly with `.Equals` requires constructing the generic type itself - https://docs.microsoft.com/en-us/dotnet/framework/reflection-and-codedom/how-to-examine-and-instantiate-generic-types-with-reflection

I suspect @bording or @stebet would know how to do that right off the top of their heads.

Reply:
Probably the best way is using Roslyn for code generation. But that means rewtiting Apigen